protocol/app/prepare/other_msgs.go

```diff
@@ -36,7 +39,12 @@ func GetGroupMsgOther(availableTxs [][]byte, maxBytes uint64) ([][]byte, [][]byt
 }
 
 // RemoveDisallowMsgs removes any txs that contain a disallowed msg.
-func RemoveDisallowMsgs(ctx sdk.Context, decoder sdk.TxDecoder, txs [][]byte) [][]byte {
+func RemoveDisallowMsgs(
+	ctx sdk.Context,
+	decoder sdk.TxDecoder,
+	extCommitCodec compression.ExtendedCommitCodec,
+	txs [][]byte,
```

Comment:
nit/alternative idea: when VE is enabled, the txs passed into this function should have already filtered out the VE txs. For example, use `txs[1:][]` to pass this function. 

the reason why i'd recommend this is so that this function remains true to its original intention of just filtering out DisallowMsgs. it seems like we are piggy backing this extra func

Reply:
this is optional and can be done later when we have more time

Reply:
The only reason I'm not doing this is because after the v5.0.0 upgrade, there is [an initial 4 blocks](https://github.com/dydxprotocol/v4-chain/blob/8454dc9dbbfe571f824a6df7b82ef07616ed508f/protocol/app/upgrades/v5.0.0/upgrade.go#L223) where VE are not enabled. During those 4 blocks, we still want to pass in all transactions. I found this logic to be simplier